orchestration/prepare.go

```diff
@@ -39,7 +50,12 @@ func RegisterPrepare() {
 }
 
 func PrepareOrchestrator(ctx context.Context, targetEnv cfg.Configuration, cluster runtime.Cluster, wait bool) (any, error) {
-	env, err := MakeOrchestratorContext(ctx, targetEnv)
+	prebuilts, err := fetchPrebuilts(ctx)
```

Comment:
Thinking about this, this means that the CLI is blocked on an API call before it can check if the orchestrator is up to date. That both means that (a) we're spending anything between 50 and 1000ms on this call (it's not edge cached), and (b) we require being online to use ns.

The orchestrator can do this type of blocking work behind the scenes. What about we attempt to connect to the current orchestrator, and ask it if it is up to date. And the orchestrator could check the version from time to time in the background.